pyscriptjs/src/interpreter_client.ts

```diff
@@ -39,7 +41,8 @@ export class InterpreterClient extends Object {
      * interface.
      * */
     async initializeRemote(): Promise<void> {
-        await this._unwrapped_remote.loadInterpreter(this.config, this.stdio);
+        const internal_js_mod = { define_custom_element, showWarning };
```

Comment:
I think this variable (and all the others down the chain) should be called `_pyscript_js`, in accordance to name passed to `registerJsModule` below.

But maybe it should be called differently: if I understand correctly, these are JS functions which are supposed to run in the main thread: so maybe `_pyscript_js_main`?